instrumented-tests/Integration/IntegrationTests/LoggingIntegrationTests.swift

```diff
@@ -4,11 +4,16 @@
  * Copyright 2019-2020 Datadog, Inc.
  */
 
-import Datadog
+@testable import Datadog
 import HTTPServerMock
 import XCTest
 
 class LoggingIntegrationTests: IntegrationTests {
+    override func setUp() {
+        super.setUp()
+        try! clearPersistedLogs()
```

Comment:
Maybe we can do it in base `IntegrationTests`? This way we won't need `@testable import Datadog` in this file, and we'll make sure we're accessing only public APIs. WDYT?